server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -1231,7 +1230,7 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 
 						this.updateNackMessages(
 							controlContents.identifier,
-							controlContents.content !== undefined ? controlContents : undefined,
+							controlContents.content === undefined ? undefined : controlContents,
```

Comment:
```suggestion
							controlContents.content ?? controlContents,
```

Reply:
That isn't the same though. The current check returns `controlContents` iff `controlContents.content` is defined, otherwise it returns `undefined`.

Reply:
Oh, misread, you're right :)